deflate_stored.c

```diff
@@ -0,0 +1,194 @@
+/* deflate_stored.c -- store data without compression using deflation algorithm
+ *
+ * Copyright (C) 1995-2013 Jean-loup Gailly and Mark Adler
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#include "zbuild.h"
+#include "deflate.h"
+#include "deflate_p.h"
+#include "functable.h"
+
+unsigned read_buf(PREFIX3(stream) *strm, unsigned char *buf, unsigned size);
```

Comment:
This should be moved to private header, so there is no risk of conflicting declarations.

Should be `Z_INTERNAL`.

Reply:
It is moved.